### ISSUE TITLE
im suprised this is actually legit

### ISSUE THREAD

**Issue by HidanOnCord** [Gucci637/Fentanyl#2]
its legit i ran it and monitered the requests and nothing was posted to nothing im suprised ty gucci

**Comment by Gucci637** [comment]
Yeah, like I said I'm hoping this will stop people from falling for fake ones.
Enjoy! :P
